code/datums/uplink_item.dm

```diff
@@ -376,24 +376,59 @@ var/list/uplink_items = list()
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 40
 
+/datum/uplink_item/dangerous/pdw
+	name = "Gepard PWD"
+	reference = "PWD"
+	desc = "A fully-loaded PWD that fires 7.62x25mm rounds with a 30-round magazine and is compatible with suppressors. Best used on armored targets."
```

Comment:
`A fully-loaded PWD that fires 7.62x25mm rounds with a 30-round magazine and is compatible with suppressors.`

This is a mouthful of a run-on sentence, I would split this up a bit. Also `PDW`.